README.md

```diff
@@ -11,6 +11,7 @@ If you would like to be notified of future conferences that would be added to th
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
 | [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on latest happenings in the field of web technologies. Hosting, Domain, Blogging and Web technologies are major discussions. |
+| [PyCon Pune](https://pune.pycon.org/2018/) | 08-11 Feb | Pune | PyCon Pune is a conference which includes first two days conference(08 - 09) and then two days dev spirint (10 - 11) to contribute to upstream projects. |
```

Comment:
That's ok - I'll squash using the Github UI. :smile: 
Just send in another commit with the change -- that would do.